pkg/pillar/cmd/domainmgr/containerd.go

```diff
@@ -36,44 +39,45 @@ type KeyValue struct {
 
 // getContainerPath return the path to the root of the container. This is *not*
 // necessarily the rootfs, which may be a layer below
-func getContainerPath(id string) string {
-	return path.Join(containersRoot, id)
+func getContainerPath(containerID string) string {
+	return path.Join(containersRoot, CtrdServicesNamespace, containerID)
```

Comment:
Something isn't quite right here. The namespace isn't precisely under `/run/containerd/<namespace>` but is split:

* the namespace itself is under `/run/containerd/runc/<namespace>`, with the container as `/run/containerd/runc/<namespace>/<container_name>/`
* the rootfs is somewhere else, pointed to by `/run/containerd/runc/<namespace>/<container_name>/state.json`, particularly the property `config.rootfs`. For example, here it is on a default docker Ubuntu install:

```
root@ip-172-31-15-153:~# cat /run/containerd/runc/default/alpine2/state.json | jq -r '.config.rootfs'
/run/containerd/io.containerd.runtime.v1.linux/default/alpine2/rootfs
root@ip-172-31-15-153:~# ls /run/containerd/io.containerd.runtime.v1.linux/default/alpine2/rootfs
bin  dev  etc  home  lib  media  mnt  opt  proc  root  run  sbin  srv  sys  tmp  usr  var
```

The location is configurable, as is the runtime config file.

Reply:
Yeah, you are right. I missed runtime in the path. Corrected that.

Reply:
The rootfs still might be in a different location. Need to read the `state.json`, or know how we will configure the containers on creation.

Reply:
Changed the rootfs location under /persist/runx/